src/stanza.c

```diff
@@ -431,7 +431,7 @@ int xmpp_stanza_to_text(xmpp_stanza_t *stanza,
     ret = _render_stanza_recursive(stanza, buffer, length);
     if (ret < 0) return ret;
 
-    if (ret > length - 1) {
+    if ((unsigned)ret > length - 1) {
```

Comment:
> What I meant is that size_t can be unsigned long instead of unsigned int or even something else

sure, that's possible (but shouldn't matter in the compare as long as the result can be held in the variable!?!)

> typedef size_t ssize_t

ew